src/components/Picker/BasePicker/index.js

```diff
@@ -15,13 +15,23 @@ class BasePicker extends React.Component {
         };
 
         this.updateSelectedValueAndExecuteOnChange = this.updateSelectedValueAndExecuteOnChange.bind(this);
+        this.executeOnCloseAndOnBlur = this.executeOnCloseAndOnBlur.bind(this);
     }
 
     updateSelectedValueAndExecuteOnChange(value) {
         this.props.onInputChange(value);
         this.setState({selectedValue: value});
     }
 
+    executeOnCloseAndOnBlur() {
+        // Picker's onClose is not executed on Web and Desktop, so props.onClose has to be called with onBlur callback.
+        this.props.onClose();
+
+        if (this.props.onBlur) {
```

Comment:
@luacmartins what do you think of making this prop's default `() => {}` so that we don't need this if check and just always call `this.props.onBlur()`? I'm not actually sure what the better code pattern is

Reply:
Yea, let's do that! Thanks @stitesExpensify!

Reply:
Good Idea! I made the changes.